scripts/lint.sh

```diff
@@ -22,7 +22,8 @@ fi
 # Check modules
 echo "Running go mod tidy/verify ..."
 go mod tidy
-git diff --exit-code -- go.mod go.sum
+git diff --exit-code -- go.mod
+git diff --exit-code -- go.sum
```

Comment:
Skusil som ten diff rozdelit, ci to pomoze.

Cielom je zisit, ze nie su ziadne necommitnute zmeny v `go.mod/go.sum`,
a u teba to funguje nejak inak (?) ako inde

Reply:
@jprochazk skusis prosim? teda oba problemy stale trvaju?

Reply:
Pořád stejný problém

Reply:
Zkusil jsem na jiném počítači (taky Windows -> WSL2 Ubuntu + Docker přímo ve WSL2), `docker system prune` + fresh clone repozitáře, stejný problém

Reply:
Navrhuju hodit to do backlogu a zatím to nechat na ubuntu. IMHO to nevadí, když je taky CI zatím na ubuntu.

Reply:
revertol som to